airbyte-integrations/connectors/source-close-com/source_close_com/close_com.yaml

```diff
@@ -0,0 +1,475 @@
+    type: DefaultPaginator
+    url_base: "*ref(definitions.requester.url_base)"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "_limit"
+    page_token_option:
+      inject_into: "request_parameter"
+      field_name: "_skip"
+    pagination_strategy:
+      class_name: "source_close_com.OffsetIncrementWorkaround"
+#      type: OffsetIncrement
+      page_size: "{{ options['items_per_page'] }}"
+  cursor_paginator:
+    type: DefaultPaginator
+    url_base: "*ref(definitions.requester.url_base)"
+    page_token_option:
+      inject_into: "request_parameter"
+      field_name: "_cursor"
+    pagination_strategy:
+      type: CursorPagination
+      cursor_value: "{{ response['cursor_next'] }}"
+      page_size: -1
+    page_size_option:
+      inject_into: request_parameter
+      field_name: ""
+  single_stream_slicer:
+    start_datetime:
+      datetime: "{{ config['start_date'] }}"
+      datetime_format: "%Y-%m-%d"
+    start_time_option:
+      field_name: "{{ options['cursor_field_name'] }}__gt"
+      inject_into: "request_parameter"
+    stream_slice_field_start: "date_created"
+    cursor_field: "{{ options['cursor_field_name'] }}"
+    datetime_format: "%Y-%m-%d"
+    class_name: "source_close_com.SingleStreamSlicer"
+  single_stream_query_slicer:
+    $ref: "*ref(definitions.single_stream_slicer)"
+    start_time_option:
+      field_name: "query"
+      inject_into: "request_parameter"
+    start_time_option_tpl: "sort:updated date_updated >= %s"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    requester:
+      $ref: "*ref(definitions.requester)"
+    paginator:
+      type: NoPagination
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    primary_key: "id"
+  full_refresh_stream:
+    $ref: "*ref(definitions.base_stream)"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      paginator:
+        $ref: "*ref(definitions.offset_paginator)"
+  full_refresh_cursor_stream:
+    $ref: "*ref(definitions.base_stream)"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      paginator:
+        $ref: "*ref(definitions.cursor_paginator)"
+  incremental_stream:
+    $ref: "*ref(definitions.full_refresh_stream)"
+    retriever:
+      $ref: "*ref(definitions.full_refresh_stream.retriever)"
+      stream_slicer:
+        $ref: "*ref(definitions.single_stream_slicer)"
+    stream_cursor_field: "date_updated"
+    $options:
+      cursor_field_name: "date_updated"
+  incremental_cursor_based_stream:
+    $ref: "*ref(definitions.incremental_stream)"
+    retriever:
+      $ref: "*ref(definitions.incremental_stream.retriever)"
+      paginator:
+        $ref: "*ref(definitions.cursor_paginator)"
+  activities_base_stream:
+    $ref: "*ref(definitions.incremental_stream)"
+    stream_cursor_field: "date_created"
+    $options:
+      primary_key: "id"
```

Comment:
the primary key should be defined outside the `options` parameter

Reply:
I can put it here as well when I need to specify it by primary_key per stream. But you are right, this key isn't used and was finally defined globally in base stream. It should be removed from here